charts/argo-rollouts/templates/dashboard/clusterrole.yaml

```diff
@@ -71,4 +71,12 @@ rules:
     verbs:
       - list
       - watch
-{{- end }}
+  - apiGroups:
```

Comment:
📝 Policy is from [here](https://github.com/argoproj/argo-rollouts/blob/master/manifests/role/argo-rollouts-clusterrole.yaml#L83-L91).
LGTM.

<img width="679" alt="image" src="https://user-images.githubusercontent.com/30188755/234014326-2e8cbe5c-f35c-4f88-87af-255da70857fc.png">